### ISSUE TITLE
Modules do not install from WebUI

### ISSUE THREAD

**Issue by treed593** [tiredofit/docker-freepbx#119]
When installing Modules via the WebUI, the Download progress shows and states the installation is successful however, the new menu options do not appear however installing via fwconsole at the CLI does work.

**Comment by selukov** [comment]
try restart apache server, it helped me
docker exec -it freepbx-app /bin/bash
/etc/init.d/apache2 restart

**Comment by selukov** [comment]
try command in docker
`fwconsole moduleadmin list`
I installed backup & filestore modules by the WebUI and WebUI shows that they are installed, but there is no WebUI specific to these modules
I executed the commands in docker:
```
docker exec -it freepbx-app /bin/bash
fwconsole moduleadmin list
```

And the WebUI specific to these modules appeared

**Comment by tiredofit** [comment]
It sounds as if with the latest changes to Freepbx something is not being refreshed/restarted. I'm curious - If you install a module via WebUI and then restart container, does it appear?

**Comment by selukov** [comment]
I have a problem with a restart, the docker freepbx-app does not see the mysql why I don’t know. I think I have error in mysql settings. 
docker-compose up/down not change the system WebUI until execute comand `fwconsole moduleadmin list`

**Comment by tiredofit** [comment]
I will take that command and make it execute upon container startup and consider making a cron entry to execute it every 15 minutes. 

The MySQL database not appearing after restart sounds something of concern - Best for a seperate issue (it may already be open) but it seems many people are having more issues than ever seen before with some of the recent updates related to the FreePBX Framework.

**Comment by selukov** [comment]
I don't think you need to add a command to cron, plugins are not often installed. I think this is a bug in WebUI. Maybe there is some WebUI API to add a command to it

**Comment by treed593** [comment]
Has anyone found a workaround for this?

**Comment by tiredofit** [comment]
Problem was related to upstream, now resolved with `tiredofit/freepbx:15-4.0.0`
